doc/whats_new/v1.1.rst

```diff
@@ -92,6 +92,9 @@ Changelog
 - |Fix| :class:`decomposition.FastICA` now validates input parameters in `fit` instead of `__init__`.
   :pr:`21432` by :user:`Hannah Bohle <hhnnhh>` and :user:`Maren Westermann <marenwestermann>`.
 
+- |Fix| :class:`decomposition.FactorAnalysis` now validates input parameters in `fit` instead of `__init__`.
+  :pr:`21713` by :user:`Haya <HayaAlmutairi>` and :user:`Krum Arnaudov <krumeto>`.
+
```

Comment:
It would be nice to reduce the length of the line here as well (even this is not like code):

```suggestion
- |Fix| :class:`decomposition.FactorAnalysis` now validates input parameters
  in `fit` instead of `__init__`.
  :pr:`21713` by :user:`Haya <HayaAlmutairi>` and
  :user:`Krum Arnaudov <krumeto>`.

```